pinot-broker/src/main/java/org/apache/pinot/broker/api/RequestStatistics.java

```diff
@@ -50,12 +50,22 @@
   private boolean _isNumGroupsLimitReached;
   private int _numExceptions;
   private String _brokerId;
+  private String _offlineBrokerTenant;
```

Comment:
```suggestion
  private String _offlineServerTenant;
```
Same foe the realtime part

Reply:
done